welcome/src/main/res/layout/welcome_on_view_cell_layout.xml

```diff
@@ -14,9 +14,12 @@
 
         <ImageView
             android:id="@+id/image"
-            android:layout_width="match_parent"
-            android:layout_height="@dimen/tourSummaryImageHeight"
+            android:layout_width="0dp"
+            android:layout_height="240dp"
             android:scaleType="centerCrop"
+            app:layout_constraintDimensionRatio="H,240:240"
```

Comment:
could use `1:1` I think if its a square since you already specify height in the layout.

Reply:
I see. Updated